src/celeritas/inp/PhysicsModel.hh

```diff
@@ -11,31 +11,79 @@ namespace celeritas
 namespace inp
 {
 //---------------------------------------------------------------------------//
-//! Seltzer-Berger bremsstrahlung model
+/*!
+ * Seltzer-Berger bremsstrahlung model.
+ *
+ * \todo Move \c sb_data from celeritas::ImportData here.
+ */
 struct SBBremsModel
 {
 };
 
-//! Relativistic bremsstrahlung model
+//---------------------------------------------------------------------------//
+/*!
+ * Relativistic bremsstrahlung model.
+ */
 struct RelBremsModel
 {
     //! Account for LPM effect at very high energies
     bool enable_lpm{true};
 };
 
-//! Muon bremsstrahlung model
+//---------------------------------------------------------------------------//
+/*!
+ * Muon bremsstrahlung model.
+ */
 struct MuBremsModel
 {
 };
 
+//---------------------------------------------------------------------------//
+// PAIR PRODUCTION MODELS
+//---------------------------------------------------------------------------//
+/*!
+ * Bethe-Heitler relativistic pair production from gammas.
+ */
+struct BHPairProdModel
+{
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * Pair production from muons.
+ *
+ * \todo Move MuPPET table celeritas::ImportMuPairProductionTable here.
+ */
+struct MuPairProdModel
+{
+};
+
+//---------------------------------------------------------------------------//
+// ALIASES
 //---------------------------------------------------------------------------//
 //!@{
 //! \name Model aliases
 //! \todo rename `em/model` to match
 
-using SeltzerBergerBremsModel = SBBremsModel;
-using RelativisticBremModel = RelBremsModel;
+#if 0
+using BetheBlochModel       = BBIoniModel;
+using BraggModel            = BraggIoniModel;
+using CombinedBremModel     = SBRelBremModel;
+using CoulombScatteringModel= WentzelScatModel;
+using EPlusGGModel          = EPlusGGModel;
+using ICRU73QOModel         = ICRU73QOModel;
+using KleinNishinaModel     = KleinNishinaModel;
+using LivermorePEModel      = LivermorePEModel;
+using MollerBhabhaModel     = MollerBhabhaModel;
+using MuBetheBlochModel     = MuBetheBlochModel;
+using RayleighModel         = RayleighModel;
+#endif
+
+using BetheHeitlerModel = BHPairProdModel;
 using MuBremsstrahlungModel = MuBremsModel;
+using MuPairProductionModel = MuPairProdModel;
+using RelativisticBremModel = RelBremsModel;
+using SeltzerBergerModel = SBBremsModel;
```

Comment:
More naming discussion:

- I slightly prefer the more descriptive names to the abbreviations for the models (`BetheBloch` to `BB`; `BetheHeitler` to `BH`; `SeltzerBerger` to `SB`), although I'm guilty of abbreviating the distributions and other helpers
- Maybe keep as `BetheBlochModel` and `BraggModel`, or to be consistent change to `MuBetheBlochIoniModel` and `MollerBhabhaIoniModel`
- `BHPairProdModel` could alternatively be `RelPairProdModel` (which it now technically is)

Reply:
I remember @pcanal way back in the day weighing in on our first model naming scheme and preferring the longer spelling. My biggest concern is consistency, then I suppose readability should be second, then writability. I think the model names start to blur a bit as we start changing them slightly to apply different corrections... RelPair is BetheHeitler with LPM so maybe we should call it LPMBHPairProd 🤪

Reply:
I prefer LPMBHPP

Reply:
🤮!

Reply:
I still lean for the longer name, they make it a bit easier for newcomers to get into the code without having to search through for the acronyms ...

Reply:
Yeah true... but since a lot of these models are effectively just "<process> optimized for <energy range> with <assumptions> by being a combination of <foo>, modified by <bar>, with <jimmy's> form factors, and <vladimir's> arbitrary tweaks"

Reply:
It is still "in the family of ..." or at least "inspired by ...." ... but more accurately it is "best numerical approximation of the XYZ model/process that we could come up with given the information we have so far (and to be fair, also with a bit of duck tape, chewing gum, self annulling pair of bugs and prayers and somehow it looks like it was passing some sort of test someday, I am sure ... maybe )"  :)

Reply:
All right, so to get this moving I'll delete these aliases for now and we can resume in a later commit :)